### ISSUE TITLE
Add missing ES6/7 Array built-ins

### ISSUE THREAD

**Issue by fatcerberus** [svaarala/duktape#1106]
- [ ] `Array.from()`
- [ ] `Array.of()`
- [ ] `Array.prototype.copyWithin()`
- [ ] `Array.prototype.entries()`
- [ ] `Array.prototype.fill()`
- [ ] `Array.prototype.find()`
- [ ] `Array.prototype.findIndex()`
- [ ] `Array.prototype.includes()`
- [ ] `Array.prototype.keys()`
- [ ] `Array.prototype.values()`
- [ ] `Array.prototype[@@toIterator]()`
- [ ] `Array.prototype[@@unscopables]`

There are more of these than I thought.

**Comment by fatcerberus** [comment]
At first I thought `Array.prototype.includes()` might be redundant since you can just use indexOf(), but there is a difference: includes() does a SameValue comparison whereas indexOf() uses strict equality (`===`).

**Comment by svaarala** [comment]
Yeah, `includes()` should be pretty straightforward to implement by tacking it onto the existing duk_bi_array_prototype_indexof_shared() helper.

**Comment by MonkeybreadSoftware** [comment]
It would be nice to have those functions added.

Maybe some can be done in JavaScript, if that's easier for now?

```
Array.prototype.includes = function(value) {
	for (i = 0; i < this.length; i++) {
		if (this[i] == value) return true;
	}
	return false;
}  
```
